src/Acuminator/Acuminator.Vsix/Resources/VSIXResource.resx

```diff
@@ -365,10 +365,10 @@ This option should be used together with the "Banned API File" setting.</value>
     <value>Enable code analysis</value>
   </data>
   <data name="Setting_CodeAnalysis_SuppressionMechanismEnabled_Description" xml:space="preserve">
-    <value>Do not display the diagnostics specified in the suppression file. If set to false, Acuminator ignores the suppression file and displays all found diagnostics.</value>
+    <value>Enables Acuminator diagnostics suppression with suppression comments and Acuminator suppression files. If this setting is set to false, Acuminator will ignore suppression comments in the source code and Acuminator suppression files and display all found diagnostics.</value>
```

Comment:
@EAndrosova please check the changes in the name and description of the VS settings that controls the Acuminator suppression mechanism